src/features/login/Login.tsx

```diff
@@ -12,6 +12,7 @@ import {Input} from '../../components/input/Input';
 import {Routes} from '../../navigation/Router';
 import {tryLogin} from '../../middlewares/loginThunk';
 import {MainStackNavigationParams} from '../../navigation/mainStackNavParams';
+import {localization} from '../../localization/i18n';
```

Comment:
we still have i18n file but we should just know that: we import localisation instance that it, no library name ...